### PR TITLE
i18n: adding first few motivational quotes and compliments in italian

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
Thanks for getting these started!

File: client/i18n/locales/italian/motivation.json [hunk 1]
```diff
@@ -1,819 +1,56 @@
...
-    "Gotta go fast!",
-    "Yipee!",
-    "Cowabunga!",
-    "Moon Prism Power!",
-    "Plus Ultra!"
+    "A tutta birra!",
+    "Pikachu, scelgo te!",
+    "Hai fegato!",
+    "Alla Batmobile",
+    "Tu sei un mago, harry",
```
Comment (nhcarrigan) [inline client/i18n/locales/italian/motivation.json:7]:
```suggestion
    "Tu sei un mago, Harry",
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Mentioned in chat.

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @ieahleen 🎉

**Pull request merged**